lms/validation/authentication/_oauth.py

```diff
@@ -1,4 +1,3 @@
-"""Validation for OAuth views."""
```

Comment:
This docstring wasn't accurate: the validation schemas for most OAuth-related views are elsewhere, this module only contains one particular schema. There's nothing useful for a docstring on this module to say.